<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="2" column="0">
     <widget class="QTableWidget" name="table_widget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>35</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <column>
       <property name="text">
        <string>Арт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Разм</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Параметр</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Штук</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Крой</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Пачка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Раскройщик</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сиб. шт.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цен. ткани</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>на ед</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>% обр</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Опер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фурн</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>% брак</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цен + %</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Кг ткани</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>+обр</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="4" rowspan="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Расчитать</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="le_art">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Размер</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_param">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Параметр</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Артикул</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_size">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
        <item>
         <widget class="QPushButton" name="pb_set">
          <property name="text">
           <string>Выбрать арт + раз + пар</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
        <item>
         <widget class="QPushButton" name="pb_art">
          <property name="text">
           <string>Выбрать арт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
        <item>
         <widget class="QPushButton" name="pb_size">
          <property name="text">
           <string>Выбрать арт + раз</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Дата</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QDateEdit" name="de_date_in">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDateEdit" name="de_date_from">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>927</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>1032</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_art</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_art()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>-98</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_size</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_art()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>-61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_set</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_art()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>-68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_calc()</slot>
  <slot>ui_view_art()</slot>
 </slots>
</ui>
